Sources/Rendering/Core/ColorTransferFunction/index.js

```diff
@@ -217,7 +217,10 @@ function vtkColorTransferFunction(publicAPI, model) {
       const before = JSON.stringify(model.nodes);
       model.nodes = nodes;
       const after = JSON.stringify(model.nodes);
-      return publicAPI.sortAndUpdateRange() || before !== after;
+      if (publicAPI.sortAndUpdateRange() || before !== after) {
+        publicAPI.modified();
+        return true;
+      }
```

Comment:
And for performance reasons, I would suggest:
```
if (model.nodes !== nodes && !vtkMath.areEquals(model.nodes, nodes)) {
  model.nodes = nodes;
  return publicAPI.sortAndUpdateRange()
}
return false;
```
WDYT @jourdain ?

Reply:
We need to be careful as some of those changes were made to prevent unnecessary update. 
I would be fine on refining the modification detection. A test would be ideal, but maybe tricky to implement?